components/ILIAS/Authentication/classes/class.ilAuthPageEditorGUI.php

```diff
@@ -153,9 +187,60 @@ private function forwardToPageObject(): void
 
     private function show(): void
     {
+        $this->context = AuthPageEditorContext::LOGIN;
+        $this->showIliasEditor();
+    }
+
+    private function showLoginEditor(): void
+    {
+        $this->context = AuthPageEditorContext::LOGIN;
+        $this->showIliasEditor();
+    }
+
+    private function showLogoutEditor(): void
+    {
+        $this->context = AuthPageEditorContext::LOGOUT;
         $this->showIliasEditor();
     }
 
+    private function handleLoginPageActions(): void
+    {
+        $action = $this->http->wrapper()->query()->retrieve(
+            'loginpage_languages_action',
+            $this->refinery->byTrying([
+                $this->refinery->kindlyTo()->string(),
+                $this->refinery->always('')
+            ])
+        );
+
+        $keys = $this->http->wrapper()->query()->retrieve(
+            'loginpage_languages_key',
```

Comment:
I suggest to rename `loginpage_languages_key` to `authpage_languages_key` (or something similar).